openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/TagRepository.java

```diff
@@ -85,6 +85,11 @@ public void setFullyQualifiedName(Tag tag) {
     }
   }
 
+  @Override
+  public String getFullyQualifiedNameHash(Tag tag) {
```

Comment:
The default implementation in EntityRepository can just hash entity.getFullyQualifiedName and override only when default behavior is different.